### PR TITLE
bump tooling versions and include gnuplot

### PR DESCRIPTION
**Pull request by dprotaso**

### CODE REVIEW

File: images/prow-tests/Dockerfile [hunk 1]
```diff
@@ -183,15 +184,15 @@ COPY images/prow-tests/in-gvm-env.sh /usr/local/bin
 FROM golang:1.17 AS external-go-gets
 
 ARG KUBETEST2_VERSION=5e5d3e9eebc6a609aa428bd3e2a1c0c3566d5baf
-ARG KIND_VERSION=v0.12.0
+ARG KIND_VERSION=v0.14.0
 ARG KO_VERSION=v0.11.2
-ARG PROTOC_GEN_GO_VERSION=v1.26.0
-ARG PROTOC_GEN_GOGOFASTER_VERSION=v1.3.1
-ARG GOTESTSUM_VERSION=v1.7.0
+ARG PROTOC_GEN_GO_VERSION=v1.29.0
```
Comment (kvmware) [inline images/prow-tests/Dockerfile:189]: Looks like latest is 1.28 https://pkg.go.dev/google.golang.org/protobuf/cmd/protoc-gen-go

File: images/prow-tests/Dockerfile [hunk 2]
```diff
@@ -183,15 +184,15 @@ COPY images/prow-tests/in-gvm-env.sh /usr/local/bin
...
-ARG GOTESTSUM_VERSION=v1.7.0
+ARG PROTOC_GEN_GO_VERSION=v1.28.0
+ARG PROTOC_GEN_GOGOFASTER_VERSION=v1.3.2
+ARG GOTESTSUM_VERSION=v1.8.1
 ARG KAIL_VERSION=v0.15.0
 ARG LICHE_VERSION=v0.0.0-20200229003944-f57a5d1c5be4
-ARG GO_LICENSES_VERSION=v1.0.0
-ARG DOCKER_CREDENTIAL_GCR_VERSION=v2.0.5
+ARG GO_LICENSES_VERSION=v1.2.1
+ARG DOCKER_CREDENTIAL_GCR_VERSION=v2.1.5
```
[the remark this answers is not in the document]
Comment (kvmware) [reply images/prow-tests/Dockerfile:195]: `$ go install github.com/GoogleCloudPlatform/docker-credential-gcr@a14b07c 
go: downloading github.com/GoogleCloudPlatform/docker-credential-gcr v1.5.1-0.20220606170751-a14b07c61269
`

[the remark this answers is not in the document]
Comment (kvmware) [reply images/prow-tests/Dockerfile:195]: Looks like their versioning is messed up and you can refer to the hash instead

[the remark this answers is not in the document]
Comment (upodroid) [reply images/prow-tests/Dockerfile:195]: Maybe grab the binary from GitHub releases and drop it in /usr/local/bin ? This package was messy to upgrade when I rewrote this container earlier this year.

[the remark this answers is not in the document]
Comment (kvmware) [reply images/prow-tests/Dockerfile:195]: Can't we just use `gcloud` or is there some reason we are using this? `Note: docker-credential-gcr is primarily intended for users wishing to authenticate with GCR in the absence of gcloud`

[the remark this answers is not in the document]
Comment (kvmware) [reply images/prow-tests/Dockerfile:195]: https://github.com/GoogleCloudPlatform/docker-credential-gcr/issues/115 for what it is worth

**Comment by dprotaso** [comment]
@upodroid to confirm you setup automatic bumping prow images when this file changes?

**Comment by upodroid** [comment]
I did.

- https://github.com/knative/test-infra/blob/main/prow/autobump-config/knative-prow-tests.yaml
- Last Execution: https://github.com/knative/test-infra/pull/3338

**Pull request merged**